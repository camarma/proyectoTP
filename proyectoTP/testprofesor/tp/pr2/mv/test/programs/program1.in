push 2
push 3
add
halt
end
