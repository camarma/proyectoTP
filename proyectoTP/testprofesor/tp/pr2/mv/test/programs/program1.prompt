step
step 5
quit