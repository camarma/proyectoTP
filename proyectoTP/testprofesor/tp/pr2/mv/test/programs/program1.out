Introduce el programa fuente
> > > > > El programa introducido es:
0: PUSH 2
1: PUSH 3
2: ADD
3: HALT
> Comienza la ejecucion de  PUSH 2
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: 2
> Comienza la ejecucion de  PUSH 3
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: 2 3
Comienza la ejecucion de  ADD
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: 5
Comienza la ejecucion de  HALT
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: 5
> 